test/libsolidity/syntaxTests/parsing/missing_parameter_name_in_named_args.sol

```diff
@@ -3,4 +3,4 @@ contract test {
 	function b() returns (uint r) { r = a({: 1, : 2, : 3}); }
 }
 // ----
-// ParserError: (143-143): Expected token Identifier got 'Colon'
+// ParserError: (143-143): Expected 'Identifier' but got ':'
```

Comment:
Also should a `expectToken` have a special case for `'Identifier` -> `identifer` ?